src/webob/request.py

```diff
@@ -752,11 +752,11 @@ def POST(self):
         Return a MultiDict containing all the variables from a form
         request. Returns an empty dict-like object for non-form requests.
 
-        Form requests are typically POST requests, however PUT & PATCH requests
-        with an appropriate Content-Type are also supported.
+        Form requests are typically POST requests, however PUT, PATCH & DELETE
+        requests with an appropriate Content-Type are also supported.
         """
         env = self.environ
-        if self.method not in ('POST', 'PUT', 'PATCH'):
+        if self.method not in ('POST', 'PUT', 'PATCH', 'DELETE'):
```

Comment:
I would argue for removal of this entirely, so that it just works on all request methods, Webdav has a bunch of other request methods for example and it doesn't make sense if we disallow `.POST` on those.

As long as we keep the line on 766 we should be good to go.

Reply:
It is a cool idea. I will remove this condition.